packages/gestalt-design-tokens/tokens/vr-theme/comp/spinner/default.json

```diff
@@ -3,17 +3,39 @@
     "spinner": {
       "color": {
         "background": {
-          "pink": {
```

Comment:
could we just do 1, 2, 3 and avoid token name changes? 

spinner-color-background-01
spinner-color-background-02
spinner-color-background-03